Procfile

```diff
@@ -1 +0,0 @@
-web: node index.js
```

Comment:
Yup, temporarily at least, because the only way to run the app now is with `node ./lib/server` and it only runs in "development" mode (watches files, etc). For now I will delete `index.js` and change the `main` on `package.json`.